tslint-sonarts.json

```diff
@@ -13,6 +13,7 @@
     "no-variable-usage-before-declaration": true,
     "no-misspelled-operator": true,
     "no-inconsistent-return": false,
-    "no-unconditional-jump": true
+    "no-unconditional-jump": true,
+    "no-misleading-array-reverse": false
```

Comment:
@stas-vilchik since we did the separation, I think rule should be activated back:)